webapp-django/bin/ci.sh

```diff
@@ -18,11 +18,12 @@ echo "Linting..."
 git ls-files crashstats | grep '\.py$' | xargs flake8 | bin/linting.py
 
 echo "Starting tests..."
+# Override configuration with better defaults for the tests
 export SECRET_KEY="doesn't matter, tests"
 export CACHE_BACKEND="django.core.cache.backends.locmem.LocMemCache"
 export CACHE_LOCATION="crashstats"
-export OLD_PYTHONPATH=$PYTHONPATH
-export PYTHONPATH=../:$PYTHONPATH
-FORCE_DB=true python manage.py test --noinput
-export PYTHONPATH=$OLD_PYTHONPATH
+# Change AWS_HOST to empty string so tests use boto.s3.connect_to_region
+export AWS_HOST=
+
+PYTHONPATH=../:$PYTHONPATH FORCE_DB=true python manage.py test --noinput
 echo "Tests finished."
```

Comment:
The tests for the symbol upload view mock boto. However, they mock the `boto.s3.connect_to_region` way of getting a connection instance rather than the `boto.connect_s3` way. Rather than adding a lot of complexity to the tests to mock both situations (Travis effectly runs with `AWS_HOST=` and Circle runs with `AWS_HOST=localstack-s3`), we set `AWS_HOST=` so all test systems test the path the server environments will be running.